docs/examples/hasura/tutorial.md

```diff
@@ -1,244 +1,29 @@
----
-title: Hasura
-sidebar_label: Hasura
----
+# KintoHub Hasura Example
 
-The [open-source project Hasura](https://hasura.io) makes it incredibly easy to setup a [GraphQL API](https://graphql.org) that has async processing patterns built-in.
+## Overview
+Hasura is a graphql engine which makes it easy to create CRUD graphql on top of postgres aside from stitching together APIs providing and eventing queue!
 
-Hasura does this by leveraging [PostgreSQL](https://www.postgresql.org) as a backing store. On top of that Hasura adds queuing and a subscription mechanism.
+## Deployment
 
-This tutorial shows how you can setup Hasura easily with KintoHub, and do that for multiple environments.
+1. Apply this template to your [Github](https://github.com/kintohub/hasura-template/generate)
+2. Create a [Website Block](https://beta.kintohub.com) on KintoHub:
+3. Set the name of your block
+4. Select `Dynamic Web App`
+5. For `language` select `Custom Dockerfile`
+6. Set the port to 8080
+7. Hit the `Create` button
+8. You're now good to build! Click `Build Latest Commit`. Once complete, Click  Now click `Add To Project` and Create a new Project.
+9. Scroll to "KintoBlocks" section and in the Search Box type "Postgres" and select it to add a Postgres Database.
+10. Click `Create New Project` at bottom right
 
-Prefer a video instead? See Hasura in action on KintoHub in this [recording of our talk at OSCON](https://www.youtube.com/watch?v=UTmCPEtxALk).
+The deployment can take up to 3 minutes. Once successful, click `Open URL` on the Hasura Block under "KintoBlocks" section.
 
-<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/UTmCPEtxALk" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+## Usage
 
-## Prepare a PostgreSQL database
+Click `Open URL` and start playing with hasura. We recommend using the advance migration functionality for multiple environments such as `dev` and `staging`.
```

Comment:
need to mention the `Public API` setting when deploying?

Reply:
I'll mention this stuff in the blog post. Also its a web block so it actually does not have a public/private, you would setup a password on it.